2.5/test/run

```diff
@@ -179,8 +191,15 @@ for server in ${WEB_SERVERS[@]}; do
   ct_npm_works
   check_result $?
 
+  test_from_dockerfile
+  check_result $?
```

Comment:
(1) One more thing that I'm unsure, feel free to correct me - we don't want to run this for every `WEB_SERVER`, right? Or is there some benefit?
_ _ _ _

(2) I have mentioned it'd be good to run it in a loop, but I didn't mean this one, but instead f.e.:

```suggestion
for file_append in '' s2i; do 
    test_from_dockerfile $file_append
    check_result $?
done
```

Which is a bit "overkill", with just two dockerfiles, so feel free to keep it un-looped.

Reply:
It's the reason why I use this way, due to two files I think it's overkill to use a loop.